VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -52,17 +60,33 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseComponent implemen
             videoCallService,
             deviceTypeService,
             router,
-            consultationService
+            consultationService,
+            userMediaService
         );
     }
 
     ngOnInit() {
         this.errorCount = 0;
         this.logger.debug('Loading judge waiting room');
+        this.showChooseDeviceDialog();
         this.connected = false;
         this.getConference().then(() => {
             this.startEventHubSubscribers();
             this.getJwtokenAndConnectToPexip();
+            this.subscribeToAcceptConsultation();
+        });
+    }
+
+    showChooseDeviceDialog() {
+        this.displayDeviceChangeModal = !this.userMediaService.getShowDialogChooseDevice();
+        this.userMediaService.updateShowDialogChooseDevice(true);
+    }
+
+    subscribeToAcceptConsultation() {
```

Comment:
I would advise closing the modal in `handleParticipantStatusChange` if the user is moved into consultation

Reply:
I close on accept